slicebase/etc/mlab/slicectrl-functions

```diff
@@ -45,12 +46,12 @@ function stop_after_everything () {
 function kill_remaining () {
     local signal=$1
     KILLED=
-    pgrep -v -f "slicectrl|rsync|rpm" > /var/run/$SLICENAME/pids.afterstop
+    pgrep -v -f "slicectrl|rsync|rpm|bash|login|/bin/su" > /var/run/$SLICENAME/pids.afterstop
     if [ -f /var/run/$SLICENAME/pids.beforestart ] ; then
         for pid in `comm -23 <( sort /var/run/$SLICENAME/pids.afterstop ) \
-                             <( sort /var/run/$SLICENAME/pids.beforestart )`; do 
-            echo "STILL RUNNING: " `ps ax | grep $pid`
-            kill -$signal $pid
+                             <( sort /var/run/$SLICENAME/pids.beforestart )`; do
+            echo "STILL RUNNING: " `ps ax | grep $pid | grep -v grep` 1>&2
+            kill -$signal $pid 2> /dev/null
             KILLED="TRUE"
```

Comment:
Should this be wrapped in something like?

`if [ $? -eq 0 ]; then`

Or can we just assume that kill did its job?